src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/ViewModels/DateTimePickerViewModel.cs

```diff
@@ -143,6 +145,24 @@ public int Second
                 Value = new DateTime(Year, Month, Day, Hour, Minute, value);
             }
         }
+
+        public string AmPm
+        {
+            get;
+            private set;
+        }
+        #endregion
+
+        #region Properties
+        protected override void OnPropertyChanged(AdvancedPropertyChangedEventArgs e)
+        {
+            base.OnPropertyChanged(e);
+
+            if (e.PropertyName == "Value")
```

Comment:
Always use string.Equals. In this case, it will go right, but if e.PropertyName would be of type "object", it would return false (so better just to always use string.Equals)